frontend/src/metabase/admin/databases/components/DatabaseEditApp/Sidebar/Sidebar.tsx

```diff
@@ -25,7 +25,9 @@ interface DatabaseEditAppSidebarProps {
   database: Database;
   isAdmin: boolean;
   isModelPersistenceEnabled: boolean;
-  updateDatabase: (database: { id: DatabaseId } & Partial<IDatabase>) => void;
+  updateDatabase: (
```

Comment:
Do you think it would be nicer to keep other signatures consistent, e.g. `deleteDatabase` for sure, also returns a promise?

I believe almost if not all other actions probably return a promise as well.

Reply:
You're right, wrapped other callback return types into `Promise` as well 👍